src/main/java/ru/admiralnsk/admiralbd/models/Departure.java

```diff
@@ -55,4 +58,17 @@
     private Integer carriageKm;
     private Integer volume;
     private Integer rate;
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        Departure departure = (Departure) o;
+        return departureDate.equals(departure.departureDate) && carriageNumber.equals(departure.carriageNumber) && documentNumber.equals(departure.documentNumber) && cargo.equals(departure.cargo);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(departureDate, carriageNumber, documentNumber, cargo);
+    }
 }
```

Comment:
`@EqualsAndHashCode(of = {"departureDate", "carriageNumber", "documentNumber", "cargo"})`
=)